CHANGELOG.md

```diff
@@ -27,6 +27,8 @@
 - AppEngine default timeout is now the same as connection timeout
 - Removes invalid function `User.create()` since creating a child-user requires a name
 - Removes deprecated class `CreditCard`, please use alternative `Billing` class
+- Removes all the getters of each object
```

Comment:
This should be clarified to say that all getters are available via lombok and aren't explicitly in the code anymore. Keeping it the way it's worded makes it sound like people cannot use getters anymore.

You should mention that all setters have been removed.

Reply:
resolved